contracts/netmap/contract.go

```diff
@@ -665,6 +780,32 @@ func filterNetmap(ctx storage.Context) []Node {
 	return result
 }
 
+func fourBytesBE(num int) []byte {
```

Comment:
this is too specific and i see the only usage of it, so suggest to leave `netmapNode2AtEpochPrefix() []byte` for now. But dont insist